model/nml/namelists.nml

```diff
@@ -0,0 +1,24 @@
+$ -------------------------------------------------------------------- $
+$ WAVEWATCH III -  model parameters namelist
+$ -------------------------------------------------------------------- $
+$ This file is to be used with ww3_grid.nml where its name is the value
+$ of GRID%NML
+$
+$ The contents is equal to the namelist input of ww3_grid.inp
+$ (traditional form) ending with a mandatory string 'END OF NAMELISTS'
```

Comment:
I think its a good idea to add in this .nml file, but in that case maybe we should start to populate this template with the namelist section included in the ww3_grid.inp template?

At the very least should probably add in the RTD namelists text?

Reply:
I have included the namelist section of the ww3_grid.inp template and the RTD namelists text. I edited the ROTB text in ww3_grid.inp to make it consistent.